assets/locales/en.json

```diff
@@ -149,5 +150,41 @@
   "cantAddIpfsToPath": {
     "title": "IPFS on PATH",
     "message": "Could not add IPFS to the PATH."
+  },
+  "launchAtLoginNotSupported": {
+    "title": "Launch at login",
+    "message": "Launch at login is not supported on your platform."
+  },
+  "launchAtLoginFailed": {
+    "title": "Launch at login",
+    "message": "Launch at login could not be enabled on your machine."
+  },
+  "ipfsOnPathInstall": {
+    "title": "Install IPFS on PATH",
+    "message": "By enabling this option, IPFS will be available on your command line as \"ipfs\". This action is reversible.",
+    "action": "Install"
```

Comment:
For both the dialog boxes (and their menu items) related to IPFS on PATH, we're using "install/enable" and "uninstall/disable" interchangeably, which is a little confusing. Let's decide on either install/uninstall or enable/disable and be consistent throughout ... I might go for enable/disable so that it feels more like an option rather than something more severe, but there might be precedence for install/uninstall elsewhere that I'm unaware of?

Reply:
Just updated everywhere to enable/disable that is not related to IPFS Desktop updates 😃 

>  I might go for enable/disable so that it feels more like an option rather than something more severe

Agreed. We're not actually installing anything but when updating or enabling `npm on IPFS`, which I will make more clear.